### PR TITLE
Enh #5224: Add reply-to email in the settings

### PR DESCRIPTION
**Pull request by funkycram**
**What kind of change does this PR introduce?** (check at least one)

Enh #5224: Add reply-to email in the settings

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [x] When resolving a specific issue, it's referenced in the PR's description (e.g. `Fix #xxx[,#xxx]`, where "xxx" is the Github issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included
- [x] Changelog was modified

If adding a **new feature**, the PR's description includes:
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: protected/humhub/modules/admin/models/forms/MailingSettingsForm.php [hunk 1]
```diff
@@ -76,6 +79,16 @@ public function attributeLabels()
         ];
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function attributeHints()
+    {
+        return [
+            'systemEmailReplyTo' => Yii::t('AdminModule.settings', 'Optional. If used, check your DKIM/SPF in the DNS records for this email address'),
```
Comment (ArchBlood) [inline protected/humhub/modules/admin/models/forms/MailingSettingsForm.php:88]: Here are a few ideas;

`Optional: If used, remember to check your DKIM/SPF in the DNS records for this email address.`
or
`Optional: If used, remember to check your DKIM/SPF for this email address in the DNS records.`

Comment (luke-) [reply protected/humhub/modules/admin/models/forms/MailingSettingsForm.php:88]: @funkycram Do we really need to address a correctly configured email server here? Might be more something for the documentation, that just confuses regular users.

I would rather write something like:
`Optional. Default reply address for system emails like notifications.`

@Semir1212 Any text idea?

**Comment by funkycram** [comment]
`Optional. Default reply address for system emails like notifications.` fits well.
If no others comments in the next days, I will change it to this.

**Comment by funkycram** [comment]
@luke- I have updated the attribute hint to `Optional. Default reply address for system emails like notifications.`.

**Comment by funkycram** [comment]
My last commit about `- Enh #5471: On the pending approval page, add grouped actions and custom columns` has been added to this PR about the `Enh #5224: Add reply-to email in the settings`. If it's a problem, I will separate them.

**Comment by luke-** [comment]
@funkycram Would be great if you could seperate the features into two PRs..

**Pull request closed without merging**